src/Text/Pandoc/Readers/LaTeX.hs

```diff
@@ -726,6 +728,14 @@ dolstinline = do
 doLHSverb :: PandocMonad m => LP m Inlines
 doLHSverb = codeWith ("",["haskell"],[]) <$> manyTill (satisfy (/='\n')) (char '|')
 
+dosiunitx :: PandocMonad m => LP m Inlines
+dosiunitx = do
+  skipopts
+  value <- char '{' >> manyTill anyChar (char '}')
```

Comment:
`braced` does not parse latex commands e.g. `\euro`. So I cannot write `\SI{1.5}{\euro}`.